.github/workflows/test.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           context: .
           push: false
           load: true
-          tags: exercism/go-test-runner
+          tags: exercism/go-representer

```

Comment:
@ErikSchierboom It seemed there was a wrong tag mentioned here.

Reply:
Great catch!